src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -140,6 +140,32 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="ImportOrder"/&gt;
+ * </pre>
+ * <p>
+ * Example of Import order:
+ * </p>
+ * <pre>
+ * import java.io.IOException; //OK
+ * import java.net.URL; //OK
+ * import java.security.KeyManagementException; //OK
+ *
+ * import javax.net.ssl.SSLContext; //OK
+ * import javax.net.ssl.TrustManager; //OK
+ * import javax.net.ssl.X509TrustManager; //OK
+ *
+ * import org.apache.http.conn.ClientConnectionManager; //OK
+ * import org.apache.http.conn.scheme.Scheme; //OK
+ *
```

Comment:
Please show some violations here, also this example probably doesn't need to have this many packages.

Reply:
ok no violations here bcoz this piece is for default importOrder values.